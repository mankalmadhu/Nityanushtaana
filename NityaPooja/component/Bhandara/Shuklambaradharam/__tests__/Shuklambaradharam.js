import 'react-native';
import React from 'react';
import ಶುಕ್ಲಾ೦ಬರಧರ೦ from '../Shuklambaradharam';

import Renderer from 'react-test-renderer';

test("ಶುಕ್ಲಾ೦ಬರಧರ೦" , () => {

expect(Renderer.create(
    <ಶುಕ್ಲಾ೦ಬರಧರ೦ />
).toJSON()).toMatchSnapshot();

});