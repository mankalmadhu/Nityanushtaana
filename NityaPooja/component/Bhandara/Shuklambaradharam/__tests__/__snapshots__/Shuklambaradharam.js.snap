// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಶುಕ್ಲಾ೦ಬರಧರ೦ 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
  style={
    Object {
      "color": "#ffffff",
      "fontFamily": "Ganapati",
      "fontSize": 18,
      "lineHeight": 50,
      "marginLeft": 12,
      "textAlign": "justify",
    }
  }
>
  ಶುಕ್ಲಾ೦ಬರಧರ೦ ವಿಷ್ಣು೦ ಶಶಿ ವರ್ಣ೦ ಚತುರ್ಭುಜ೦ | ಪ್ರಸನ್ನ ವದನ೦ ಧ್ಯಾಯೇತ್ ಸರ್ವ ವಿಘ್ನೋಪ ಶಾ೦ತಯೇ ||
</Text>
`;
